apps/agile-project-fe/src/app/components/input/input.tsx

```diff
@@ -1,17 +1,33 @@
 import styled from 'styled-components';
 
 export interface InputProps {
-  variant: 'email';
+  variant: 'text' | 'email' | 'password';
 }
 
-const StyledInput = styled.div`
-  color: pink;
-`;
+const StyledInput = styled.div``;
+
+export function Input({ variant }: InputProps) {
+  const inputType =
+    variant === 'text'
+      ? 'text'
+      : variant === 'email'
+      ? 'email'
+      : variant === 'password'
+      ? 'password'
+      : 'default';
```

Comment:
Is this really necessary? We're just returning exactly the string we receive

you can define a default value for variant in the props object like so

```ts
export function Input({ variant = "default" }: InputProps) {
```